ppr-ui/src/composables/mhrRegistration/useHomeOwners.ts

```diff
@@ -112,6 +122,17 @@ export function useHomeOwners (isMhrTransfer: boolean = false) {
     }
   }
 
+  const hasETAGroupError = (group: MhrRegistrationHomeOwnerGroupIF): boolean => {
+    // If a group contains multiple ETA's within one group, return invalid
+    if (group.owners.filter(owner => executorTrusteeAdmin.includes(owner.partyType)).length > 1) {
+      return true
+    } else
+    // If group has a mix of ETA and living owner, return invalid
+    if (group.owners.some(owner => executorTrusteeAdmin.includes(owner.partyType)) &&
+        group.owners.some(owner => !executorTrusteeAdmin.includes(owner.partyType))) {
+      return true
+    } else return false
+  }
```

Comment:
I believe you can simplify this to ..
`return (group.owners.filter(owner => executorTrusteeAdmin.includes(owner.partyType)).length > 1) ||`
`(group.owners.some(owner => executorTrusteeAdmin.includes(owner.partyType)) &&
        group.owners.some(owner => !executorTrusteeAdmin.includes(owner.partyType)))(group.owners.some(owner => executorTrusteeAdmin.includes(owner.partyType)) &&
        group.owners.some(owner => !executorTrusteeAdmin.includes(owner.partyType)))`
        
        
        Though doesn't the second condition always resolve as false? They look mutually exclusive no, as in if one is true the other is false and vice versa?

Reply:
What were the findings here?

Reply:
They are mutually exclusive as you suggested. Scrapped that function and replaced in most recent commit

Reply:
oh right on, yes looks good, nice work 👍